pkg/cmd/server/origin/controller/build.go

```diff
@@ -7,8 +7,7 @@ import (
 
 	builddefaults "github.com/openshift/origin/pkg/build/admission/defaults"
 	buildoverrides "github.com/openshift/origin/pkg/build/admission/overrides"
-	buildclient "github.com/openshift/origin/pkg/build/client"
-	buildcontrollerfactory "github.com/openshift/origin/pkg/build/controller/factory"
+	buildcontroller "github.com/openshift/origin/pkg/build/controller/build"
```

Comment:
this is part of @deads2k's recent change to the build controller. I believe this is a pattern we want to following going forward?